### ISSUE TITLE
Better approach for Makefile

### ISSUE THREAD

**Issue by MichaelChirico** [MichaelChirico/r-ci-samples#5]
e.g. `facebook/prophet` has `./Makefile` and `./docs/Makefile`; it appears to be random which will be included --> bigger diff from auto-update.

Not sure a robust way to fix this ATM

**Comment by JanCaha** [comment]
It could be done in the same way as github actions. Create a folder for each repo and store the makefiles there. The path to file can be turned to a long file name. So your examples would be `Makefile` and `docs_Makefile` stored in `prophet` folder. 

I believe that this makes sense for files where more than file can be expected (e.g. `Makefile`).

**Comment by MichaelChirico** [comment]
yes that's basically what i was thinking. still feels a bit inelegant... and i would lean towards leaving the full path within the `prophet` folder:

```
./prophet
  Makefile
  doc/
    Makefile
```
